diff --git a/dist/index.mjs b/dist/index.mjs
index 984a2f0063eb0a1af50dda3102c416ecbee4efac..3568390553c22475998adfb55dff3df856a117e5 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -5,9 +5,9 @@ import path from 'node:path';
 import { createHash } from 'node:crypto';
 
 function resolveCompiler(root) {
-  const compiler = tryRequire("vue/compiler-sfc", root) || tryRequire("vue/compiler-sfc");
+  const compiler = tryRequire("vue2/compiler-sfc", root) || tryRequire("vue2/compiler-sfc");
   if (!compiler) {
-    throw new Error(`Failed to resolve vue/compiler-sfc.
+    throw new Error(`Failed to resolve vue2/compiler-sfc.
 @vitejs/plugin-vue2 requires vue (>=2.7.0) to be present in the dependency tree.`);
   }
   return compiler;